src/enyo-samples/lib/BackgroundTaskManagerSample/BackgroundTaskManagerSample.css

```diff
@@ -0,0 +1,57 @@
+.background-task-manager-sample {
+	background: linear-gradient(to bottom, #9be2fe 0%,#67d1fb 100%);
+	position: absolute;
+	top: 0;
+	bottom: 0;
+	left: 0;
+	right: 0;
+}
+
+.background-task-manager-sample .block {
+	position: absolute;
+	border: 2px solid black;
+	width: 33%;
+	text-align: center;
+}
+
+.background-task-manager-sample .tower .block.important {
+	border: none;
+	background: white;
+	opacity: 0.25;
+}
+
+.background-task-manager-sample .left .block {
+	left: 0;
+	right: 66%;
+	background: linear-gradient(red, #f06d06);
+}
+
+.background-task-manager-sample .middle .block {
+	left: 33%;
+	right: 33%;
+	background: linear-gradient(to right, rgba(73,155,234,1) 0%, rgba(32,124,229,1) 100%);
+}
+
+.background-task-manager-sample .right .block {
+	left: 66%;
+	right: 0;
+	background: linear-gradient(to right, rgba(241,231,103,1) 0%, rgba(254,182,69,1) 100%);
+}
+
+.background-task-manager-sample .info {
+	position: fixed;
+	background-color: #999;
+	color: #FFF;
+	padding: 10px;
+	border: 1px solid #777;
+	border-radius: 5px;
+}
+
+.tower-popup {
```

Comment:
Can't specify this as `.background-task-manager-sample .tower-popup` as this is in the floating layer. We'll need to figure out a better way to handle this for Strawman for floating layer elements in general. One idea is to automatically add a class specific to the loaded sample to the body element.